Handdeck/Source/Handdeck.cpp

```diff
@@ -0,0 +1,68 @@
+#include "Handdeck.hpp"
+
+HandDeck::HandDeck(int _startSize)
+:
+startSize(_startSize)
+{
+
+}
+
+TableDeck::TableDeck(int _tableSize)
+:
+tableSize(_tableSize)
+{
+
+}
+
+DiscardDeck::DiscardDeck(int _maxSize)
+:
+maxSize(_maxSize)
+{
+
+}
+
+
+PullDeck::PullDeck(int _maxSize)
+:
+maxSize(_maxSize)
+{
+
+}
+
+void Deck::addCard(Card cardToAdd)
+{
+  Cards.push_back(cardToAdd);
+}
+
+void HandDeck::drawCard(PullDeck &pullDeck)

```

Comment:
Co jeżeli przeciwnik użyje czaru, który podgląda 5 pierwszych kart z Decku?

Wasza implementacja losuje kartę, więc to źle działa.